### PR TITLE
CMake: only build regression_fixture for testing

### PR DESCRIPTION
**Pull request by havogt**

### CODE REVIEW

**Comment by havogt** [comment]
launch strgrid

File: src/tools/CMakeLists.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
-add_library(regression_main regression_fixture.cpp)
-target_link_libraries(regression_main gtest gridtools)
+if(ENABLE_TESTING)
+  add_library(regression_main regression_fixture.cpp)
+  target_link_libraries(regression_main gtest gridtools)
```
Comment (lukasm91) [inline src/tools/CMakeLists.txt]: I think we have a mixed convention, but if I add new CMake code, I use four spaces. Can we agree on a convention?

Comment (havogt) [reply src/tools/CMakeLists.txt]: 4 is find with me. I used the convention from directory up.

**Comment by havogt** [comment]
launch strgrid

**Pull request merged**